frontend/src/components/AppBar.js

```diff
@@ -2,14 +2,16 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import MuiAppBar from '@material-ui/core/AppBar';
-import Button from '@material-ui/core/Button';
 import Grid from '@material-ui/core/Grid';
 import Toolbar from '@material-ui/core/Toolbar';
 import Typography from '@material-ui/core/Typography';
 import { withStyles } from '@material-ui/core/styles';
 
 import SearchForm from './SearchForm';
 
+// The location for a redirected search request
+const SEARCH_LOCATION = '/';
```

Comment:
When we sort out routing in #100 or #97, I think we'll need to re-factor this into a separate ``/search`` route and turn the AppBar's search box into something which re-directs. I've opened #102 to track this.